<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerClasesDisponibles</class>
 <widget class="QWidget" name="VerClasesDisponibles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clases Disponibles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tablaClases"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnApuntarse">
     <property name="text">
      <string>Apuntarse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnVolver">
     <property name="text">
      <string>Volver</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
